mscore/file.cpp

```diff
@@ -2155,6 +2155,12 @@ bool MuseScore::savePdf(Score* cs_, QPrinter& printer)
                         p.fillRect(pl.at(n)->pageBoundingRect(), preferences.getColor(PREF_UI_CANVAS_FG_COLOR));
                         p.drawRect(pl.at(n)->pageBoundingRect());
                         }
+                  else {
+                        QPixmap* pm = new QPixmap(preferences.getString(PREF_UI_CANVAS_FG_WALLPAPER));
+                        p.drawTiledPixmap(pl.at(n)->pageBoundingRect(),
+                                          *pm,
+                                          pl.at(n)->pageBoundingRect().topLeft() - QPoint(lrint(mscore->currentScoreView()->matrix().dx()), lrint(mscore->currentScoreView()->matrix().dy())));
```

Comment:
Add one more line break there before the 4th arg.

Reply:
There is no 4th argument, but I see that it is quite convoluted. It should be clearer now.

Reply:
Hm... the third argument might be both useless and an obstacle in the future, I'll remove it.